Clarify that subscription matching does not count PAYG instances (#46)
uyuni-project/uyuni-docs: modules/client-configuration/pages/vhm.adoc

If your VMs are provided by different zones, even if they are within the same region, you might not meet the conditions of the ``1-2 Virtual Machines`` subscription.
In this case, check your subscription carefully.

[NOTE]
====
For BYOS instances (bring your own subscription), all installed products are passed to the subscription matcher.
If your public cloud instances are PAYG (pay as you go), their base products will be excluded from the subscription matcher counting.

> The calculation about whether an instance is PAYG or BYOS is done at the time of registration or when a "Hardware Refresh" action is exectuted.

Reviewer: ```suggestion
The calculation about whether an instance is PAYG or BYOS is done at the time of registration or when a hardware refresh action is executed.
```

Author: Thanks @Loquacity :+1:

If your VMs are provided by different zones, even if they are within the same region, you might not meet the conditions of the ``1-2 Virtual Machines`` subscription.
In this case, check your subscription carefully.

[NOTE]
====

> Only BYOS (Bring-Your-Own-Subscription) instances are passed to the subscription matcher. In case your Public Cloud instances are PAYG (Pay-As-You-Go), these will be excluded from the subscription matcher counting.

Reviewer: ```suggestion
Only instances using bring your own subscription (BYOS) are passed to the subscription matcher. 
If your public cloud instances are pay as you go (PAYG), they will be excluded from the subscription matcher counting.
```

Reviewer 2: I'm not that happy with your suggestion ;-)  I assume that these instances are prominently labeled either BYOS or PAYG, thus it might make sense to have this in the running text. This is a tough reading for the international readers, IMO:
> If your public cloud instances are pay as you go ...
I'll provide a suggestion, too ;)

Reviewer: Sure :) I was mostly unhappy about the hyphens and initcaps, and then decided to turn it around.

> Only BYOS (Bring-Your-Own-Subscription) instances are passed to the subscription matcher. In case your Public Cloud instances are PAYG (Pay-As-You-Go), these will be excluded from the subscription matcher counting.

Reviewer 2: ```suggestion
Only BYOS instances (bring your own subscription) are passed to the subscription matcher.
If your public cloud instances are PAYG (pay as you go), they will be excluded from the subscription matcher counting.
```
